afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -27,9 +37,15 @@
 
     private final WeakListenerList<AppStorageListener> listeners;
 
-    public NodeEventClient(String fileSystemName, WeakListenerList<AppStorageListener> listeners) {
+    private final RemoteServiceConfig config;
+
+    private int reconnectionInterval;
```

Comment:
Why is this needed here? Couldn't this reconnectionInterval be just left local to the onClose method just like the counter ?

Reply:
It's done